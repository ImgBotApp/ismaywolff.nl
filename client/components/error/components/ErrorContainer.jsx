import styled from 'styled-components'
import { colors } from '../../../styles'

const ErrorContainer = styled.div`
  background: ${colors.lightRed};
  color: ${colors.darkRed};
  fill: ${colors.darkRed};
  border-radius: 0.15em;
  padding: 0.5rem 1rem;
`

export default ErrorContainer
