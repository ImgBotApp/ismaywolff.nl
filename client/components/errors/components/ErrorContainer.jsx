import { PropTypes } from 'react'
import glamorous from 'glamorous'

const ErrorContainer = glamorous.div(props => ({
  background: props.background,
  color: props.color,
  fill: props.color,
  borderRadius: '0.15em',
  padding: '0.5rem 1rem'
}))

ErrorContainer.propTypes = {
  background: PropTypes.string.isRequired,
  color: PropTypes.string.isRequired
}

export default ErrorContainer
