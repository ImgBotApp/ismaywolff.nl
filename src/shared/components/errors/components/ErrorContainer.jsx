import { string } from 'prop-types'
import styled from 'styled-components'

const ErrorContainer = styled.div`
  background: ${props => props.background};
  border-radius: 0.15em;
  color: ${props => props.color};
  fill: ${props => props.color};
  padding: 0.5rem 1rem;
`

ErrorContainer.propTypes = {
  background: string.isRequired,
  color: string.isRequired
}

export default ErrorContainer
