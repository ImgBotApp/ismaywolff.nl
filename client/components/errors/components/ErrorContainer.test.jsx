import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import ErrorContainer from './ErrorContainer'

describe('<ErrorContainer />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ErrorContainer background={'green'} color={'blue'} />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
