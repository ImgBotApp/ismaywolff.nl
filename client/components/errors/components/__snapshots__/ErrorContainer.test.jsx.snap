// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorContainer /> renders correctly 1`] = `
<div
  className="css-qjaeoj"
/>
`;
